style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.header {
+  width: 100%;
+  height: 200px;
+  background-color: #88c8bc;
+  display: flex;
+  justify-content: center;
+  font-size: xx-large;
+}
+.logo {
+  padding-right: 200px;
+}
+.header .nav-bar {
+  /* border: 2px black solid; */
+  position: relative;
+  top: 50px;
+  width: 70%;
+  display: flex;
+  /* justify-content: space-around; */
+}
+.header .nav {
+  /* border: 2px blue solid; */
+  width: 40%;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+.search {
+  height: 25px;
+  border-radius: 10px;
+  padding-right: 50px;
+}
+.fa-shopping-cart {
+  padding-right: 300px;
+  padding-left: 50px;
+}
+.fa-search {
+  /* padding-right: 100px; */
+  background-color: #3f9b8a;
+  height: 35px;
+  width: 35px;
+  border-radius: 20px;
+}
+.filter {
+  position: absolute;
+  top: 80px;
+  font-size: large;
+  display: flex;
+  width: 8%;
+  justify-content: space-between;
+  padding-left: 1100px;
+}
+.choice {
+  padding-top: 30px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+.choice .button1 {
+  font-size: xx-large;
+  background-color: #88c8bc;
+  border-radius: 20px;
+  margin-right: 30px;
+}
+.choice .button2 {
+  font-size: xx-large;
+  background-color: #88c8bc;
+  border-radius: 20px;
+}
+
+.container {
+  width: 90%;
+  height: 400px;
+  display: flex;
+  justify-content: center;
+  justify-content: space-around;
+  padding-top: 50px;
+  padding-right: 200px;
```

Comment:
**Remove** ` padding-right: 200px;` its make a problem in the view of the design , its gives 200px margin from the right in this way the  images doesn't have the right position also I have noticed that you give the `width: 90%;` it will be better if you give it `width: 100%;`, because you wont the Container fill the screen not 90% of it.